app/assets/stylesheets/components/_loading-spinner.scss

```diff
@@ -1,6 +1,7 @@
 .app-c-loading-spinner {
   @include govuk-focusable;
   text-align: center;
+  padding: 200px 0;
```

Comment:
Can we use the [design system spacing scale](https://design-system.service.gov.uk/styles/spacing/) for a 'more consistent' spacing around the page? `padding: govuk-spacing(8) * 2` or something. I would ask @soniaturcotte about this too.

Reply:
Sorry - still learning to use that. Fixed :-). I'd quite like to get this in and leave any further changes to the product/testing phase - is that OK with you?

Reply:
Yep